clients/gtest/tpmv_gtest.yaml

```diff
@@ -0,0 +1,156 @@
+---
+include: rocblas_common.yaml
+include: known_bugs.yaml
+
+Definitions:
+  - &small_matrix_size_range
+    - { M:    -1, stride_a:       1 }
+    - { M:     1, stride_a:       1 }
+    - { M:     0, stride_a:       1 }
+    - { M:     1, stride_a:       1 }
+    - { M:    10, stride_a:       100 }
+    - { M:    1, stride_a:       1 }
+    - { M:    10, stride_a:      55 }
+    - { M:   100, stride_a:    40000 }
+
+  - &medium_matrix_size_range
+    - { M:   300, stride_a:   160000 }
+    - { M:   600, stride_a:   400000 }
+
+  - &large_matrix_size_range
+    - { M:  1000, stride_a:  1000000 }
+    - { M:  2000, stride_a:  4000000 }
+    - { M:  4011, stride_a: 16088200 }
+    - { M:  8000, stride_a: 64000000 }
+
+  - &incx_range
+    - [2, -2,1,-1,0,3,10]
+
+
+Tests:
+- name: tpmv_bad_arg
+  category: pre_checkin
+  function: tpmv_bad_arg
+  precision: *single_double_precisions
+  uplo: [L, U]
+  transA: [N, T, C]
+  diag: [N, U]
```

Comment:
Seems like these aren't used so could be deleted until we determine if we will improve the invalid value type checking pattern.

Reply:
Can be kept too. I'm keeping it.